src/openrct2/OpenRCT2.cpp

```diff
@@ -11,10 +11,10 @@
 
 StartupAction gOpenRCT2StartupAction = StartupAction::Title;
 utf8 gOpenRCT2StartupActionPath[512] = { 0 };
-utf8 gCustomUserDataPath[MAX_PATH] = { 0 };
-utf8 gCustomOpenRCT2DataPath[MAX_PATH] = { 0 };
-utf8 gCustomRCT1DataPath[MAX_PATH] = { 0 };
-utf8 gCustomRCT2DataPath[MAX_PATH] = { 0 };
+u8string gCustomUserDataPath = {};
+u8string gCustomOpenRCT2DataPath = {};
+u8string gCustomRCT1DataPath = {};
+u8string gCustomRCT2DataPath = {};
```

Comment:
The initializers are technically not needed, right? A string is well-formed without having to default initialize it.
```suggestion
u8string gCustomUserDataPath;
u8string gCustomOpenRCT2DataPath;
u8string gCustomRCT1DataPath;
u8string gCustomRCT2DataPath;
```

Reply:
I’m not sure and I wouldn’t want to risk it :sweat_smile:

Reply:
Tulio is correct, `std::string` is in a valid state when initialized without specifying any constructor.